app/src/main/java/com/anytypeio/anytype/ui/onboarding/OnboardingFragment.kt

```diff
@@ -873,7 +915,7 @@ class OnboardingFragment : Fragment() {
         private const val ONBOARDING_SPACE_PARAM = "space"
         private const val ONBOARDING_STARTING_OBJECT_PARAM = "startingObject"
 
-        private const val ONBOARDING_NAME_PARAM = "name"
+        private const val ONBOARDING_NAME_PARAM = "startingObject"
```

Comment:
should be name or starting object? the relation between the value and constant name is confusing.